ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -138,6 +160,7 @@
   :configurator         - a function called with the Jetty Server instance
   :async?               - if true, treat the handler as asynchronous
   :async-timeout        - async context timeout in ms (defaults to 0, no timeout)
+  :async-timeout-handler   - an async handler to handle an async contex timeout
```

Comment:
Some additional spaces here that aren't necessary:

```
:async-timeout-handler - an async handler to handle an async contex timeout
```

Also can you ensure the other keyword descriptions line up?